Remove references to postgresql-evr (#3167)
theforeman/foreman-documentation: guides/doc-Release_Notes/topics/katello.adoc

[id="katello-headline-features"]
== Headline Features

=== postgresql-evr extension no longer required

Installation of the Katello database on remote systems where root access is not available is now possible.

> Only a basic PostgreSQL installation is required, which should enable installs on systems like Amazon RDS or Azure Database for PostgreSQL.

Reviewer: This is out of scope, but we have a guide https://docs.theforeman.org/nightly/Deploying_Project_on_AWS/index-katello.html and using Amazon RDS would be a good thing to document, after it's been verified.

Reviewer 2: @maximiliankolb Just FYI, this might be useful to know with regards to your plans for the AWS guide.

> Only a basic PostgreSQL installation is required, which should enable installs on systems like Amazon RDS or Azure Database for PostgreSQL.

Reviewer 2: ```suggestion
Only a basic PostgreSQL installation is required.
With this feature, you can now install on systems like Amazon RDS or Azure Database for PostgreSQL.
```
I'm just trying to come up with a way to avoid using "should", which can evoke uncertainty and that's not really what we want :)

Author: Well, there is uncertainty since we haven't tested these external databases. If it's better to leave the mention out until it's tested, I can remove the bits mentioning "Amazon RDS or Azure Database".

Author: I'll reword it by keeping out the uncertainty, but also not saying yet that it's a fully-supported feature.

Author: Eh, but we already use "like", so we're not fully committing to saying one system is tested. I'll just use the requested changes :)

> With this feature, you can now install on systems like Amazon RDS or Azure Database for PostgreSQL.

Reviewer: Note to self: since Pulp 3.22 uses the HStore extension (which is why you need `postgresql-contrib`). It's on both  https://docs.aws.amazon.com/AmazonRDS/latest/PostgreSQLReleaseNotes/postgresql-extensions.html and https://learn.microsoft.com/en-us/azure/postgresql/extensions/concepts-extensions-versions#hstore so I'd expect it to indeed work.

> The contents of postgresql-evr have been moved into Katello itself as a migration, which enables this new capability.

Author: `postgresql-evr` was a postgresql database extension, which required installation as root. Now, we're shipping the contents of that postgresql database extension within the database migration files that live in Katello's codebase. Due to that, users no longer need to be root to enable the `evr` postgresql type.

If it might be confusing for users, I can keep line 11 out. I added it just for a little extra information.

> The contents of postgresql-evr have been moved into Katello itself as a migration, which enables this new capability.

Author: I'll remove it.

[id="katello-headline-features"]
== Headline Features

> Remote installation of the Katello database on systems where root access is not available is now possible.

Reviewer: We always add headers, so something like
```suggestion

=== postgresql-evr extension no longer required

Installation of the Katello database on remote systems where root access is not available is now possible.
```

And perhaps also note that you only need a basic PostgreSQL installation now, which means it probably also works with managed databases like Amazon RDS and others.
